app/js/components/manageApps/ManageApps.jsx

```diff
@@ -427,6 +430,13 @@ export default class ManageApps extends React.Component {
     });
   }
 
+
+  handleAlert(){
+    this.setState({
+      isOpenAlert : !this.state.isOpenAlert
```

Comment:
Why not have `true` in place of ` !this.state.isOpenAlert`

Reply:
Yah I could do that but it would mean I have to create two methods one for opening the modal and one for closing the modal, but by using `isOpenAlert: !this.state.isOpenAlert,`  it means I will just have to reuse one method to either set `isOpenAlert`  to `true` or `false`.

Reply:
@justmesam  Why would you have to create two methods? I notice that `isOpenAlert` is already set to false in state. Setting the state to true should actually just work the same.